src/Poseidon/Janno.hs

```diff
@@ -41,19 +41,23 @@ module Poseidon.Janno (
     decodingOptions,
     explicitNA,
     removeUselessSuffix,
+    parseCsvParseError,
+    renderCsvParseError,
     getMaybeJannoList
 ) where
 
 import           Poseidon.Utils                       (PoseidonException (..),
                                                        PoseidonIO, logDebug,
-                                                       logError,
+                                                       logError, logWarning,
                                                        renderPoseidonException)
 
 
 import           Control.Applicative                  (empty)
 import           Control.Exception                    (throwIO)
 import           Control.Monad                        (unless, when)
+import qualified Control.Monad.Except                 as E
```

Comment:
I think `Control.Monad.Except` and `Control.Monad.Writer` are from the `mtl` library, right? I think I prefered the `transformers` library in the past, and we might want to check in the future whether we can try and stick to one of them. But OK for now.

Reply:
At the moment we **only** depend on `mtl` and not `transformers` in poseidon-hs. I didn't change any dependencies for this PR.

Reply:
I'm actually pretty indifferent here, because I don't know how `transformers` works. I heard people are using "effect-systems" instead of monad stacks nowadays. I'm not there yet... :grimacing: